operator/src/main/java/oracle/kubernetes/operator/helpers/BasePodStepContext.java

```diff
@@ -160,6 +162,16 @@ protected List<V1EnvVar> createEnv(AuxiliaryImage auxiliaryImage, String name) {
     return vars;
   }
 
+  protected V1ResourceRequirements createResources() {
+    V1ResourceRequirements resources = getServerSpec().getResources();
+    V1ResourceRequirements resourceRequirements = null;
+    if (!resources.getLimits().isEmpty() || !resources.getRequests().isEmpty()) {
```

Comment:
Why do they both have to be non-empty? Isn't it a valid use case to have one but not the other?

Reply:
The check is to see if any one of them is non-empty (i.e. either limits is non-empty or requests is non-empty). Are you suggesting to only set the limits or requests value based on which is one is non-empty?

Reply:
Yeah, I think that I misread it, so just want to make sure. I think that it's valid to have a resources section that only configures limits or that only configures requests. Anthony's question is probably the right one... Would this fail if one of the values was configured and then where the other was null.

Reply:
ok. I have pushed a change to separate the two checks in [b8ae10d](https://github.com/oracle/weblogic-kubernetes-operator/pull/3028/commits/b8ae10deb3a77aaa0144000f69b3dfdc73ebae5e). My understanding is that setting the limits/requests value to `null` is same as not setting it.